rebasehelper/specfile.py

```diff
@@ -346,6 +347,13 @@ def _comment_out_patches(self, patch_num):
                         patch_num.remove(num)
                         break
 
+    def renew_paths_to_patches(self):
+        # Fix paths in rebase_spec_file to patches to current directory
+        for index, line in enumerate(self.spec_content):
```

Comment:
I'd still like to see some additional check like `if line.startswith('Patch'):` here.